pkg/shell/hosts_dialog.jsx

```diff
@@ -823,7 +831,7 @@ class ChangeAuth extends React.Component {
             {statement}
             <br />
             {(offer_login_password || offer_key_password) &&
-                <Form isHorizontal>
+                <Form isHorizontal onSubmit={handle_submit(callback)}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19018-20230626-125250-b763fa4b-fedora-38-devel-networking//Coverage/lcov/github-pr.diff.gcov.html)